site/docs/components/stepped-tracker/index.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Stepped tracker
+title: SteppedTracker
 data:
   description: "`SteppedTracker` visually communicates a user’s progress through a linear process. It gives the user context about where they are in the process, indicating the remaining steps and the state of all steps. It can communicate new information, errors, warnings or successful completion of a process or task."
```

Comment:
Right bar of the page is much wider than rest of site, needs to fix. I don't know why, just note here

Reply:
Done!